addons/bus/static/src/im_status_service.js

```diff
@@ -4,7 +4,9 @@ import { browser } from "@web/core/browser/browser";
 import { registry } from "@web/core/registry";
 import { timings } from "@bus/misc";
 
-export const UPDATE_BUS_PRESENCE_DELAY = 60000;
+export const UPDATE_BUS_PRESENCE_DELAY = 600000;
+export const UPDATE_BUS_PRESENCE_JITTER = 600000;
+
```

Comment:
what's the unit? could we add a comment to specify the duration in a human readable way?
```suggestion
export const UPDATE_BUS_PRESENCE_DELAY = 60 * 10000;
export const UPDATE_BUS_PRESENCE_JITTER = 60 * 10000;

```